ldk/javascript/src/network/index.ts

```diff
@@ -56,24 +106,25 @@ export interface Network {
    * @returns A promise resolving with the decoded text
    */
   decode(encodedValue: Uint8Array, encoding: string): Promise<string>;
-}
 
-const mapToHttpResponse = (response: OliveHelps.HTTPResponse) => ({
-  statusCode: response.statusCode,
-  body: new Uint8Array(response.body),
-  headers: response.headers,
-});
+  /**
+   *
+   * @param socketConfiguration a configuration object defines websocket
```

Comment:
I would almost add some kind of extra documentation here that calls out "only one listener can be added to each socket" as well.

Reply:
The comment is added on the listenMessage method, listenMessage method is not required to be used which makes this information being specific to a listenMessage method:
`  /**
   * allows to listen for a websocket message (there should be only 1 listen message callback per socket to allow messages being fully received)
   * @param callback receives text or data message from websocket and error if occurs
   */
  listenMessage: (`